automerge-c/test/ported_wasm/basic_tests.c

```diff
@@ -0,0 +1,1735 @@
+#include <float.h>
+#include <setjmp.h>
+#include <stdarg.h>
+#include <stddef.h>
+#include <stdint.h>
+#include <stdlib.h>
+#include <string.h>
+
+/* third-party */
+#include <cmocka.h>
+
+/* local */
+#include "automerge.h"
+#include "../stack_utils.h"
+
+/**
+ * \brief default import init() should return a promise
+ */
+static void test_default_import_init_should_return_a_promise(void** state);
+
+/**
+ * \brief should create, clone and free
+ */
+static void test_create_clone_and_free(void** state) {
+    AMresultStack* stack = *state;
+    /* const doc1 = create()                                                 */
```

Comment:
These commented out lines are the original typescript tests right? I personally find that they make it harder to read the test, what's the reasoning behind leaving them in?

Reply:
> These commented out lines are the original typescript tests right?

@alexjg, right.

> I personally find that they make it harder to read the test...

If you use an IDE you may be able to easily make those comments disappear from view like the [Hide Comments](https://marketplace.visualstudio.com/items?itemName=eliostruyf.vscode-hide-comments) plug-in for VSCode does.

> [W]hat's the reasoning behind leaving them in?

I left commented-out lines of TypeScript in the last few basic WASM API tests that I ported and, after realizing how useful they are for correlating WASM API usage with its corresponding C API usage, I spent many hours painstakingly weaving commented-out TypeScript into the majority of the ported basic tests and all of the ported sync tests.

@orionz, what's your opinion regarding the usefulness of the inline TypeScript code?